### ISSUE TITLE
Allow users to right click in brave://rewards-internals logs 

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#16915]
## Description

A user can highlight text in the rewards-internals logs but cannot right click (in order to open the mouse menu to copy). However, they can still use the CTRL or CMD + C shortcut to copy. 

Some users are not as comfortable with keyboard shortcuts. 

## Solution

Enable right clicking in rewards-internals logs.

**Comment by deepxcode** [comment]
HI, I am ready to work on this issue, can you assign this to me? Also, can you tell me where I can locate the code?

**Comment by jonathansampson** [comment]
@deepxcode I believe you're looking for [components/brave_rewards/resources/internals/components/log.tsx](https://github.com/brave/brave-core/blob/master/components/brave_rewards/resources/internals/components/log.tsx).

**Comment by bsclifton** [comment]
@deepxcode assigned! You'll want to check out the file @jonathansampson shared - let us know if you run into any problems building the code (getting the browser compiling can definitely be a challenge itself 😄)

**Comment by deepxcode** [comment]
@bsclifton Do I need to clone brave-core too? And do I need to install chromium code too in my device?

**Comment by bsclifton** [comment]
@deeppandya when you clone `brave-browser` and run `npm run init`, it'll download all of that (brave-core and Chromium) for you 😄

**Comment by deepxcode** [comment]
Oh okay, Thanks @bsclifton! Also, you mentioned the wrong user😅, My user name is @deepxcode.
Also, will it run fine on my laptop? Like I have a laptop with 8Gb ram, i5 8th gen, 128Gb SSD, and 1TB hard disk? or I have to upgrade my laptop?

**Comment by bsclifton** [comment]
Apologies @deepxcode- @deeppandya is my teammate Deep; GitHub kept auto-picking that from the dropdown 😄 

Your spec sounds OK! The build will use a decent amount of disk space and it'll take a while to build actually. But please give it a go- let us know how things go

**Comment by deepxcode** [comment]
Oh okay😅
@bsclifton, I am facing this error, first was facing with SSH key, I resolved that. But now, when I am `running npm run init` it shows the below error
![image](https://user-images.githubusercontent.com/31823246/127149742-082053c6-e7bf-4225-9902-11b0ee653e46.png)

**Comment by deeppandya** [comment]
@deepxcode try to delete everything inside brave_browser folder. And clone the repo again. When npm run init fails, you would see this error as it would keep stale copy of few items.

**Comment by deepxcode** [comment]
I tried this, but I am getting the same error again and again

**Comment by deeppandya** [comment]
@deepxcode did you follow the instructions here : https://github.com/brave/brave-browser/wiki/Linux-Development-Environment ?

**Comment by deepxcode** [comment]
No, I have a Windows device so I followed this documentation :- https://github.com/brave/brave-browser/wiki/Windows-Development-Environment

**Comment by deepxcode** [comment]
Update: I am able to resolve the error now. The error was thrown because I had a different version of npm installed. I reverted npm to an older version and now everything is downloading. Thanks to @bsclifton and this issue [#13631](https://github.com/brave/brave-browser/issues/13631).

**Comment by deepxcode** [comment]
@bsclifton Is it necessary to clone the repository in `C:/`. Yesterday, I was installing the software and in the end, I just ran out of C Drive storage. So asking that, if it's alright to clone the repository in another drive?

**Comment by suveshmoza** [comment]
Can I work on this issue?

**Comment by deepxcode** [comment]
Sure, you can! I will just unassign the issue to me.

**Comment by suveshmoza** [comment]
@bsclifton Getting this error after running npm run init
![Screenshot from 2021-08-24 14-51-33](https://user-images.githubusercontent.com/56023604/130591906-07ba8a94-6a25-4ba6-adf0-b3476b6290e6.png)
OS:Ubuntu 20.04
NodeJS: 14.17.5
NPM:6.14.14

**Comment by deepxcode** [comment]
@suveshmoza, I too faced the same error but on windows. I reverted back my npm to an older version like 6.14.11 and the issue was resolved.

**Comment by suveshmoza** [comment]
Thanks @deepxcode but adding --scripts-prepend-node-path solved the issue for me.

**Comment by bsclifton** [comment]
Hi folks - apologies for missing all the convo here; thanks for the PR, @suveshmoza 😄 Will check this out

**Comment by stephendonner** [comment]
Verified `PASSED` using 


Brave | 1.30.56 Chromium: 93.0.4577.58 (Official Build) beta (x86_64)
-- | --
Revision | c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS | macOS Version 11.5.2 (Build 20G95)

Steps:

1. New profile
2. Launched Brave
3. Enabled Rewards
4. Visited `brave://rewards-internals`
5. Went to `Logs` tab
6. Clicked `Refresh`
7. Selected some log lines with the mouse
8. Right clicked
9. Verified regular context menu shows up (ex: copy, paste, etc) and that you can copy the line to the clipboard

example | example
---------|---------
<img width="1200" alt="Screen Shot 2021-08-30 at 2 05 06 PM" src="https://user-images.githubusercontent.com/387249/131405652-39136026-e077-45f3-89ae-18b9b7630098.png"> | <img width="1312" alt="Screen Shot 2021-08-30 at 2 05 13 PM" src="https://user-images.githubusercontent.com/387249/131405662-2c127c94-2751-4cc7-9035-da39792d11e7.png">
